tfrog-motordriver/communication.c

```diff
@@ -818,70 +819,77 @@ static inline int data_analyze_(
     {
       static unsigned char rawdata[16];
       int data_len;
-
       if (to == id)
       {
-        data_len = decord(line, len - 1, rawdata, 16);
-        if (data_len < 6)
+        if (line[0] == PARAM_heartbeat)
```

Comment:
should be handled in https://github.com/T-frog/tf-2md3-firmware/blob/f9666a8655c0627d5fa409f4530eaea4484ada10/tfrog-motordriver/communication.c#L1584-L1622

Reply:
https://github.com/seqsense/sq_diagnostics/pull/95#discussion_r262775002

Reply:
The purpose of bypassing the decoding was to reduce data size. If you wanna keep the same way for heartbeat, yes I can fix it.